osutil/disks/disks_linux.go

```diff
@@ -188,6 +188,24 @@ var diskFromDeviceName = func(deviceName string) (Disk, error) {
 	return diskFromUdevProps(deviceName, "name", props)
 }
 
+func rootMountPointsForPartition(part Partition) ([]string, error) {
+	mounts, err := osutil.LoadMountInfo()
+	if err != nil {
+		return nil, err
+	}
+
+	mountpoints := []string{}
+	for _, mnt := range mounts {
+		if mnt.DevMajor == part.Major && mnt.DevMinor == part.Minor {
```

Comment:
Could this be a single line: `if mnt.DevMajor == part.Major && mnt.DevMinor == part.Minor && mnt.Root == "/" {` ? Or is it easier to read this way?

Reply:
I suppose it could be that way yes

Reply:
No need to change was mostly curious if there is a deeper reason for writing it like this :)

Reply:
done in https://github.com/snapcore/snapd/pull/10872